set.mm

```diff
@@ -717097,6 +717100,491 @@ Graph theory (extension)
+      VRVVDYFXKYGYHYIYJYBYLUVCUVFUVGUVH $.
+
+    $d a b g f x y $.  $d e x y $.  $d A x y $.  $d B x y $.  $d E x y $.
+    $d K x y $.  $d V x y $.  $d W x y $.
+    $( Lemma 1 for ~ isomuspgr .  (Contributed by AV, 29-Nov-2022.) $)
+    isomuspgrlem1 $p |- ( ( ( ( ( A e. USPGraph /\ B e. USPGraph )
+               /\ f : V -1-1-onto-> W ) /\ ( g : E -1-1-onto-> K
+               /\ A. e e. E ( f " e ) = ( g ` e ) ) ) /\ ( a e. V /\ b e. V ) )
+                   -> ( { ( f ` a ) , ( f ` b ) } e. K -> { a , b } e. E ) ) $=
+      ( vx wcel wa wceq wi vy cuspgr wf1o cima cfv wral cpr ccnv simpl ad2antlr
+      cv f1ocnvdm sylan wrex cupgr uspgrupgr adantr ad4antr upgredg eleq1 com12
+      biimpd imp w3a wb ad6antlr simpr ad5ant12 3jca f1ocnvfvb syl imaeq2 fveq2
+      eqeq12d rspccv adantl eqeq1 eqcoms wfn fnimapr eqeq1d wo fvex preq12b weq
+      f1ofn wf1 f1of1 anim12ci f1veqaeq syl2anr anim12d impcom orcd ex olcd vex
+      jaoi syl6ibr sylbi expcom expd imp31 eleq1d impancom sylbid mpdan sylbird
+      mpd rexlimdvva ) AUBQZBUBQZRZHIDUKZUCZRZFGEUKZUCZXNCUKZUDZXSXQUEZSZCFUFZR
+      ZRZJUKZHQZKUKZHQZRZRZYFXNUEZYHXNUEZUGZGQZYFYHUGZFQZYKYORZYNXQUHUEZFQZYQYK
+      XRYOYTYDXRXPYJXRYCUIZUJFGYNXQULUMYRYTRZYSPUKZUAUKZUGZSZUAHUNPHUNZYQYRAUOQ
+      ZYTUUGXMUUHXOYDYJYOXKUUHXLAUPUQURYSFAHPUALNUSUMUUBUUFYQPUAHHUUBUUCHQZUUDH
+      QZRZRZUUFYQUULUUFRUUEFQZYQUULUUFUUMYTUUFUUMTYRUUKUUFYTUUMUUFYTUUMYSUUEFUT
+      VBVAUJVCUULUUMUUFYQUULUUMRZUUFUUEXQUEZYNSZYQUUNXRUUMYOVDUUPUUFVEUUNXRUUMY
+      OYDXRXPYJYOYTUUKUUMUUAVFUULUUMVGYKYOYOYTUUKUUMYKYOVGVHVIFGUUEYNXQVJVKUUNX
+      NUUEUDZUUOSZUUPYQTUULUUMUURYEUUMUURTZYJYOYTUUKYDUUSXPYCUUSXRYBUURCUUEFXSU
+      UESXTUUQYAUUOXSUUEXNVLXSUUEXQVMVNVOVPVPURVCUUNUURRUUPUUQYNSZYQUURUUPUUTVE
+      ZUUNUVAUUOUUQUUOUUQYNVQVRVPUUNUUTYQTUURUUNUUTUUCXNUEZUUDXNUEZUGZYNSZYQUUN
+      UUQUVDYNUUNXNHVSZUUIUUJVDZUUQUVDSUULUVGUUMUULUVFUUIUUJXOUVFXMYDYJYOYTUUKH
+      IXNWFVFUUKUUIUUBUUIUUJUIZVPUUKUUJUUBUUIUUJVGZVPVIUQHUUCUUDXNVTVKWAUULUVEU
+      UMYQUULUVERZUUMYQUVJUUEYPFUUBUUKUVEUUEYPSZYRUUKUVEUVKTZTZYTYKUVMYOYEYJUVM
+      XOYJUVMTXMYDXOYJUUKUVLYJUUKRZXOUVLUVEUVNXORZUVKUVEUVBYLSZUVCYMSZRZUVBYMSZ
+      UVCYLSZRZWBZUVOUVKTUVBUVCYLYMUUCXNWCUUDXNWCYFXNWCYHXNWCWDUWBUVOPJWEZUAKWE
+      ZRZPKWEZUAJWEZRZWBZUVKUVRUVOUWITUWAUVRUVOUWIUVRUVORUWEUWHUVOUVRUWEUVOUVPU
+      WCUVQUWDXOHIXNWGZUUIYGRUVPUWCTUVNHIXNWHZYJYGUUKUUIYGYIUIZUVHWIHIUUCYFXNWJ
+      WKXOUWJUUJYIRUVQUWDTUVNUWKYJYIUUKUUJYGYIVGZUVIWIHIUUDYHXNWJWKWLWMWNWOUWAU
+      VOUWIUWAUVORUWHUWEUVOUWAUWHUVOUVSUWFUVTUWGXOUWJUUIYIRUVSUWFTUVNUWKYJYIUUK
+      UUIUWMUVHWIHIUUCYHXNWJWKXOUWJUUJYGRUVTUWGTUVNUWKYJYGUUKUUJUWLUVIWIHIUUDYF
+      XNWJWKWLWMWPWOWRUUCUUDYFYHPWQUAWQJWQKWQWDWSWTVAXAXBUJVCUQUQXCXDVBXEXFUQXF
+      XGXHXEXIWOXJXIXGWO $.
+
+    ${
+      $d F e x $.  $d X e x $.
+      isomuspgrlem2.g $e |- G = ( x e. E |-> ( F " x ) ) $.
+      $( Lemma 1 for ~ isomuspgrlem2 .  (Contributed by AV, 29-Nov-2022.) $)
```

Comment:
Numbering of lemmas does not agree it comments and in labels (same below).

Reply:
Why not? There are 5 lemmas for lemma isomuspgrlem2, named isomuspgrlem2a (lemma 1 ), isomuspgrlem2b (lemma 2), etc.

Reply:
The numbering of lemmas is often "flattened", that is, they are numbered from lem1 to lemX, even if, say, Lemmas 1 and 2 are only used in Lemma 3. At least, that's what I say in the main part.